sdk/python/feast/infra/online_stores/contrib/cassandra_online_store/cassandra_online_store.py

```diff
@@ -225,14 +231,25 @@ def _get_session(self, config: RepoConfig):
             return self._session
         if not self._session:
             # configuration consistency checks
-            hosts = online_store_config.hosts
             secure_bundle_path = online_store_config.secure_bundle_path
-            port = online_store_config.port or 9042
+            port = (
```

Comment:
No scylladb specific code is needed. This a configuration driven. you can delete this.